keras_contrib/applications/densenet.py

```diff
@@ -178,7 +178,7 @@ def DenseNet(input_shape=None, depth=40, nb_dense_block=3, growth_rate=12, nb_fi
 def DenseNetFCN(input_shape, nb_dense_block=5, growth_rate=16, nb_layers_per_block=4,
                 reduction=0.0, dropout_rate=0.0, weight_decay=1E-4, init_conv_filters=48,
                 include_top=True, weights=None, input_tensor=None, classes=1, activation='softmax',
-                upsampling_conv=128, upsampling_type='upsampling'):
+                upsampling_conv=128, upsampling_type='deconv'):
```

Comment:
Because it matches the paper. Also, I'm getting slight better results with it than with upsampling or subpixel.